packages/qdrant-loader/src/qdrant_loader/core/qdrant_manager.py

```
@@ -205,6 +205,75 @@ def _ensure_client_connected(self) -> QdrantClient:
+        created_indexes = []
+        failed_indexes = []
+
+        for field_name, field_schema in indexes_to_create:
+            try:
+                client.create_payload_index(
+                    collection_name=self.collection_name,
+                    field_name=field_name,
+                    field_schema=field_schema,  # type: ignore
+                )
+                created_indexes.append(field_name)
+                self.logger.debug(f"Ensured payload index for field: {field_name}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)